brain/python/arrayHelpers.h

```diff
@@ -30,24 +30,37 @@ namespace brain
 
 void importArray();
 
+/** @return True is the python object is a numpy.ndarray, false otherwise. */
```

Comment:
Yes, but why didn't you see the latest changes? Also most comments from Daniel are outdated by they still appear.